samples/CpsExtension/CpsExtension.Vsix/CpsExtension.Vsix.csproj

```diff
@@ -14,14 +14,19 @@
   <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

Comment:
I'm wondering if we still need to set the props above this early for the VSSDK.

Reply:
I tried simplifying it but ran into all kinds of problems. In earlier iteration I removed the import to the VSSDK.targets which caused the .vsix to no longer be produced. Also, it looks like it needs to follow the Sdk.targets in order to work.

Also, I tried removing some of the properties defined at the beginning of the project, but that caused other problems.

So, it sounds like until VSSDK improves the support for SDK-style projects, there is not much more we can do on the sample side.